test/skimNtuple_HHbtag.cpp

```diff
@@ -5337,7 +5408,7 @@ int main (int argc, char** argv)
 			  // Apply PUjetID only to jets with pt < 50 GeV (https://twiki.cern.ch/twiki/bin/view/CMS/HiggsToTauTauWorkingLegacyRun2#Jets)
 			  if (PUjetID_WP > -1) //PU jet ID WP = 2: loose
 				{
-				  if ( !(CheckBit(theBigTree.jets_PUJetIDupdated_WP->at(iJet), PUjetID_WP)) && tlv_additionalJet.Pt()<50.) continue;
+				  if ( !(CheckBit(theBigTree.jets_PUJetIDupdated_WP->at(iJet), PUjetID_WP)) && tlv_additionalJet.Pt()<50) continue;
 				}
 
 			  // get up/down uncertainty for this additional jet
```

Comment:
A few lines below this is the following, which should probably be year-dependant too (2.5 for 2017/18):
``` 
// Central jets ( |eta| <= 2.4 ) that pass b-jet selection
 if (TMath::Abs(tlv_additionalJet.Eta()) <= 2.4)
```

Reply:
this may be beyond the scope of this PR, but possibly a small bug fix to add to the "To do" list

Reply:
I agree it is beyond the scope of the PR. Addressed by #312.